app/views/admin/applications/show.html.erb

```diff
@@ -0,0 +1,28 @@
+<h1>Admin Application</h1>
+
+Applicant Name: <%= @application.name %>
+<br>Status: <%= @application.status %>
+<br>Description: <%= @application.description %>
+<br>Full Address: <%= @application.street_address %>, <%= @application.city %>, <%= @application.state %> <%= @application.zip_code %>
+
+<br>Pets applied for:
+<% @pet_applications.each do |pet_app| %>
+  <div id="pet_<%= pet_app.pet_id %>_buttons">
+    <p><%= pet_app.pet.name %> <%= pet_app.status %></p>
+    <% if pet_app.status.nil? %>
+      <%=form_with url:"/admin/pet_applications/#{pet_app.id}", method: :patch, data: { turbo: false }, local: true do |f| %>
+        <%= f.hidden_field :status, value: "Approved" %>
+        <%= f.hidden_field :application_id, value: pet_app.application_id %>
+
+        <%= f.submit "Approve" %> 
+      <% end %>
+
+      <%=form_with url:"/admin/pet_applications/#{pet_app.id}", method: :patch, data: { turbo: false }, local: true do |f| %>
+        <%= f.hidden_field :status, value: "Rejected" %>
+        <%= f.hidden_field :application_id, value: pet_app.application_id %>
+
+        <%= f.submit "Reject" %> 
+      <% end %>
+    <% end %>
+  </div>
+<% end %>
```

Comment:
There's a bitch of HTML missing in this view. Any test that we have should be wrapped in some sort of HTML tag, whether that be a header, paragraph, etc is up to you. We don't want tags just handing around all willy-nilly.

I also see that you've used forms in place of `button_to`... any reason? While `button_to` does transform into a form on the page in the browser, we encourage you to use the proper HTML in the view code. Looking over this code I imagine it is because you want to a couple of variables in hidden fields. This goes back to what I mentioned in the `routes.rb` file about nested routing. This eliminates the need for passing the application ID in a hidden field.

Here are a couple of ways you could have done this:

Query params
```erb
<%= button_to "Approve", "/admin/applications/#{@application.id}/pets/#{pet_app.pet.id}?status=Approved", method: :patch %>
<%= button_to "Reject", "/admin/applications/#{@application.id}/pets/#{pet_app.pet.id}?status=Rejected", method: :patch %>
```

Adding to param hash directly
```erb
<%= button_to "Approve", "/admin/applications/#{@application.id}/pets/#{pet_app.pet.id}", params: { status: "Approved" }, method: :patch %>
<%= button_to "Reject", "/admin/applications/#{@application.id}/pets/#{pet_app.pet.id}", params: { status: "Rejected" }, method: :patch %>
```